environment-dev.yml

```diff
@@ -41,3 +41,5 @@ dependencies:
       - modin-spreadsheet>=0.1.1
       - tqdm
       - git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
+      - black==22.1.0
+      - flake8==4.0.1
```

Comment:
but the actual CI recipe [doesn't use any of these](https://github.com/prutskov/modin/blob/00af9ab95c5c7b2ca9859e86d48dc7d9b1fba981/.github/workflows/ci.yml#L37)

Reply:
not relevant